nautobot/extras/views.py

```diff
@@ -1165,6 +1165,17 @@ def get_required_permission(self):
     def get_extra_context(self, request, instance):
         job_result = instance.get_latest_sync()
 
+        if job_result is None:
+            job_result = {
+                "completed": "—",
+                "date_created": "—",
+                "date_done": "—",
+                "duration": "—",
+                "name": "—",
+                "status": "N/A",
+                "user": "—",
+            }
```

Comment:
Probably should be an empty dictionary here and use the `placeholder` template tag in the template for these fields? For example `{{ result.duration | placeholder }}`

```suggestion
            job_result = {}
```

Reply:
Done. This worked fine, thanks.